pkg/webui/console/components/device-data-form/messages.js

```diff
@@ -22,6 +22,9 @@ export default defineMessages({
   deviceIdDescription: 'Unique Identifier of your device; this cannot be changed afterwards',
   deviceNameDescription: 'Human friendly name of your device for display purposes',
   deviceDescDescription: 'Optional device description; can also be used to save notes about the device',
+  nsServerAddressDescription: 'The address of the device\'s Network Server',
```

Comment:
I'm afraid this wording might be misleading, should we rather speak of `host` / `host name`? `address` sounds more like FQDN. (Same issue for gateway creation and linking form)

Reply:
This was already removed as per @bafonins' suggestion